src/js/control-bar/picture-in-picture-toggle.js

```diff
@@ -26,23 +26,17 @@ class PictureInPictureToggle extends Button {
    */
   constructor(player, options) {
     super(player, options);
-    this.on(player, ['enterpictureinpicture', 'leavepictureinpicture'], (e) => this.handlePictureInPictureChange(e));
-    this.on(player, ['disablepictureinpicturechanged', 'loadedmetadata'], (e) => this.handlePictureInPictureEnabledChange(e));
 
-    this.on(player, ['loadedmetadata', 'audioonlymodechange', 'audiopostermodechange'], () => {
-      // This audio detection will not detect HLS or DASH audio-only streams because there was no reliable way to detect them at the time
-      const isSourceAudio = player.currentType().substring(0, 5) === 'audio';
+    // Removes the pictureInPictureToggle in browsers that do not support the Picture-in-Picture API.
+    if (!('exitPictureInPicture' in document)) {
+      this.dispose();
```

Comment:
An alternative solution would be to
```javaScript
      this.player_.ready(() => {
        this.parentComponent_.removeChild(this);
        this.dispose();
      });
```

This would avoid having a component not accessible in the `controlBar.children`